tests/src/init.rs

```diff
@@ -42,7 +42,15 @@ pub fn initialize_instance(backends: wgpu::Backends, force_fxc: bool) -> Instanc
             dx12: wgpu::Dx12BackendOptions {
                 shader_compiler: dx12_shader_compiler,
             },
-            gl: wgpu::GlBackendOptions::from_env_or_default(),
+            gl: wgpu::GlBackendOptions {
+                fence_behavior: if cfg!(target_family = "wasm") {
+                    wgpu::GlFenceBehavior::AutoFinish
```

Comment:
where in the tests are we relying on this, what would break if we use `Normal`. Plz add comment